intelli.py

```diff
@@ -0,0 +1,49 @@
+from bs4 import BeautifulSoup
+import requests
+from models import Coffee
+from google.appengine.api import urlfetch
+urlfetch.set_default_fetch_deadline(45)
+
+
+def scrape_intelli():
+    roaster = 'Intelligentsia'
+    intelli = 'http://www.intelligentsiacoffee.com/products/coffee'
+    r = requests.get(intelli)
+    soup = BeautifulSoup(r.content)
+
+    # each coffee under class="grid_4 node node-type-product-coffee node-teaser build-mode-teaser""
+    coffees_for_sale = soup.find_all('div', {'class': 'node-type-product-coffee'})
+    for item in coffees_for_sale:
+        name, price, description, notes, region, status, size, product_url = [""] * 8
+        product_url = 'http://www.intelligentsiacoffee.com' + item.a['href']
+        notes_list = item.p.contents
+        notes = notes_list[2] + ', ' + notes_list[4] + ', ' + notes_list[6]
+        name = item.find('div', {'class': 'productListingDescBox'}).strong.string
+        print product_url
```

Comment:
can you use the logging stuff instead?

https://cloud.google.com/appengine/articles/logging